guide/.vuepress/branches.js

```diff
@@ -5,7 +5,7 @@ export default [
 		aliases: ['12', 'stable'],
 	},
 	{
-		label: 'v11.5',
+		label: 'v11.6',
```

Comment:
The rest are v11, shouldn't it be here too?
```suggestion
		label: 'v11',
```

Reply:
One could argue for `v11.x` like for the v12 branch. I'm going with what was discussed in the Discord, but open to suggestions. This is what is shown in the version drop-down selector at the top.

Reply:
since this PR chances the links to point to the v11 tag i think having the selector reflect that is a good idea. `v11` is the naming of the selector on the library docs as well after all

Reply:
Should the v12 selector be renamed for consistency as well? To `v12 (stable)`

Reply:
Done in the next commit